LanternExtractor/EQ/Wld/DataTypes/RenderInfo.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections.Generic;
+using System.IO;
+using GlmSharp;
+using LanternExtractor.EQ.Wld.Fragments;
+using LanternExtractor.Infrastructure;
+
+namespace LanternExtractor.EQ.Wld.DataTypes
+{
+    public class RenderInfo
+    {
+        public int Flags { get; set; }
+        public int Pen { get; set; }
+        public float Brightness { get; set; }
+        public float ScaledAmbient { get; set; }
+        public BitmapInfoReference SimpleSpriteReference { get; set; }
+        public UVInfo UVInfo { get; set; }
+        public List<vec2> UVMap { get; set; }
+
+        public static RenderInfo Parse(BinaryReader reader, List<WldFragment> fragments)
+        {
+            var renderInfo = new RenderInfo
+            {
+                Flags = reader.ReadInt32()
+            };
+
+            var ba = new BitAnalyzer(renderInfo.Flags);
+
+            var hasPen = ba.IsBitSet(0);
```

Comment:
Do we know yet what this does? See it in the later WLD files, too.

Reply:
There are a couple functions suggesting something related to palette & transparency but I haven't figured out how it all comes together yet. `t3dSetRenderContextTransparentPen` `t3dPaletteClosestPenToColor`